meetings/2023-08-02.md

```diff
@@ -0,0 +1,103 @@
+  * All - worth people looking at - <https://openjs-foundation.slack.com/archives/C027PSG2PJR/p1690225162828989>
+  * Moshe, .env support is really a different feature than the support for NODE_OPTIONS, configuring options for Node.js is a separate concern
+  * Antoine - agree its a separate concern, don’t see point of waiting.
+  * Matteo, may be an argument about not supporting anything that applies to Node.js.
+    * Geoffrey but then that is the same as .env
+  * Michael but if whole point is to configure Node.jh how does it makes sense to land without NODE_OPTIONS  and other Node environment
+  * Geoffrey, any concerns about continuing work to to figure out the NODE_OPTIONS support and then re-discuss once we figure that out
+    * nobody raised any concerns.
+  * Colin, no strong opinion on supporting environment variables, but NODE_OPTIONS is a bad way from configuring, configuring through command line flags is the better way
+
+* esm: add import.meta.dirname and import.meta.filename [#48740](https://github.com/nodejs/node/pull/48740)
+  * Antoine, ask is to add properties to import.meta. Would only be on the file protocol and would be the path and filename.  From my perspective ESM should be using URLs, strange to have properties that are only available for one protocol
+  * Geoffrey, agree with Antoine but that is not necessarily the blocker for me, but block is that this should be standardized and agreed in WinterCG before this would be added. Past experience is that if not standardized, will cause issues later on. Would be fine to add them once WinterCG agrees on a standard - <https://github.com/wintercg/proposal-common-minimum-api/issues/50>
+  * Matteo - have been one of the people who have been frustrated by this. Discussions have not been productive. Seems like people believe that it should be agnostic or that it needs to be aware of how/where it runs. What I like about Node.js is that you have that affinity with the file system. In the past it was not clear how we could provide the info. It’s a philosophical discussion so that’s why called vote as unlikely to get agreement. Getting into WinterCG is a separate discussion. Change does not remove anything from anybody not using it.
+  * Yagiz, Benjamin + 1 to what Matteo mentioned.
+  * Michael, the lack of this information has been raised as one of the blockers in moving to ESM in the latest Node.js survey.
+  * From WinterCG viewpoint, should be treated like X (missed it), we should treat things in global scope which import.meta is. Should be treated as SemVer major.  Already plan to add `import.meta.url` and `import.meta.resolve` to the minimum. That said, ok to add the bit of metadata being proposed, but ask would be some write up to WinterCG to get input to avoid other runtimes from doing something different. So coordination would be good.
```

Comment:
@jasnell can you review please?
```suggestion
  * James – From WinterCG viewpoint, runtimes are well within their rights to add anything they want to `import.meta`, but maybe we should treat things in global scope which `import.meta` is. Should be treated as SemVer major.  The WinterCG already plans to add `import.meta.url`, `import.meta.resolve`, and `import.meta.main` to the minimum common API. That said, ok to add the bit of metadata being proposed, but ask would be some write up to WinterCG to get input to avoid other runtimes from doing something different. So coordination would be good.
```